e2e/simple-crud.spec.ts

```diff
@@ -69,6 +69,22 @@ describe('Simple CRUD works', () => {
     expect(response.tag.name).toEqual(allTags[0].name);
   });
 
+  it('Remove one tag', async () => {
```

Comment:
great but can you add get by the removed id should returns in a not found error.